test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -318,7 +320,26 @@ def send_emoji(self, emoji_short_name: str, message: str):
     def verify_last_message_is_not_loaded(self):
         [loaded, _] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
         verify_false(loaded, "Success: No message was found")
-                
+
+    def verify_last_message_is_reply(self, message: str):
+        last_message_obj = self.get_message_at_index(0)
+        last_message_reply_details_obj = getChildrenWithObjectName(last_message_obj, ChatItems.STATUS_MESSAGE_REPLY_DETAILS.value)[0]
+        verify(not is_null(last_message_reply_details_obj), "Checking last message is a reply: " + message)
+
+    def verify_last_message_is_reply_to(self, reply: str, message: str):
+        last_message_obj = self.get_message_at_index(0)
+        last_message_reply_details_obj = getChildrenWithObjectName(last_message_obj, ChatItems.STATUS_MESSAGE_REPLY_DETAILS.value)[0]
+        text_message_obj = getChildrenWithObjectName(last_message_reply_details_obj, ChatItems.STATUS_MESSAGE_REPLY_DETAILS_TEXT_MESSAGE.value)[0]
+        verify_text_contains(str(text_message_obj.messageDetails.messageText), str(message))
+
+    def verify_last_message_is_reply_to_loggedin_user_message(self, reply: str, message: str):
+        last_message_obj = self.get_message_at_index(0)
+        last_message_reply_details_obj = getChildrenWithObjectName(last_message_obj, ChatItems.STATUS_MESSAGE_REPLY_DETAILS.value)[0]
+        text_message_obj = getChildrenWithObjectName(last_message_reply_details_obj, ChatItems.STATUS_MESSAGE_REPLY_DETAILS_TEXT_MESSAGE.value)[0]
+        verify_text_contains(str(text_message_obj.messageDetails.messageText), str(message))
+        verify_values_equal(str(last_message_reply_details_obj.replyDetails.sender.id), str(last_message_obj.senderId), "Message sender ID doesn't match reply message sender ID")
```

Comment:
@noeliaSD I've intrtoduced this API here to check whether the message we're replying to is indeed our own message.

Reply:
Very cool!! :)